sentry-sidekiq/lib/sentry/sidekiq/error_handler.rb

```diff
@@ -40,7 +42,13 @@ def retry_limit(context)
         when Integer
           limit
         when TrueClass
-          ::Sidekiq.options[:max_retries] || 25
+          max_retries =
```

Comment:
I haven't looked at this before, but why does sentry have to deal with the retry logic at all?

Reply:
Because with `config.sidekiq.report_after_job_retries = true`, the SDK should wait for the retry quota exhausted before reporting.